armory/utils/metrics.py

```diff
@@ -431,6 +431,192 @@ def _check_object_detection_input(y, y_pred):
         )
 
 
+def _intersection_over_union(box_1, box_2):
+    """
+    Assumes format of [y1, x1, y2, x2]
```

Comment:
I believe this will still work as intended if the input format is instead (x1, y1, x2, y2).  Is that correct?

Reply:
correct; updating documentation